.github/workflows/push-dev-to-release.yml

```diff
@@ -0,0 +1,30 @@
+name: Create Pull Request to Push Dev to Release
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: 30 17,19 * * *
```

Comment:
I think this cron schedule will run at 1730 AND at 1930. You can just use one or the other

Reply:
cool, I'll just use 1730 UTC then 👍